composer.json

```diff
@@ -20,6 +20,9 @@
         "php": ">=7.0",
         "apereo/phpcas": "^1.3",
         "phpmailer/phpmailer": ">=6.1.6",
+        "doctrine/orm": "^2.5",
```

Comment:
We removed this on pull request #154. Is it useful ?

Reply:
Works without it, safe to remove.